### PR TITLE
Workflow chapter in Reference

### PR DESCRIPTION
**Pull request by jmleroux**

### CODE REVIEW

File: reference/workflow/index.rst [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Collaborative workflow
+======================
+
+This chapter describes describes the collaborative workflow between Mary and Julia.
```
Comment (MarieMinasyan) [inline reference/workflow/index.rst]: one describes en enough :)

Comment (rybus) [inline reference/workflow/index.rst]: twice "describes"

Comment (rybus) [inline reference/workflow/index.rst]: between => involving (just being picky :p )

**Comment by pierallard** [comment]
Weird line in the 2nd draft of this picture: reference/workflow/img/partial-approve.png
GTM

**Comment by phaseinducer** [comment]
GTM

File: reference/workflow/index.rst [hunk 2]
```diff
@@ -0,0 +1,12 @@
+Collaborative workflow
+======================
+
+This chapter describes the collaborative workflow involving Mary and Julia.
+
+**These cookbooks are about a feature only provided in the Enterprise Edition.**
```
Comment (SamirBoulil) [inline reference/workflow/index.rst]: "These cookbooks describe a feature provided in the Enterprise Edition only" ?

**Pull request merged**